src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -45,7 +45,7 @@ export type UnsavedIncidentRule = {
   aggregation: AlertRuleAggregations;
   aggregations: AlertRuleAggregations[];
   projects: string[];
-  environment: string[];
+  environment: string | string[]; // Temporarily can be either
```

Comment:
Oops I missed this. Since you have this compat shim in place I'll switch over after you've merged.

Reply:
I think we needed it anyway, otherwise things would just break when your thing landed.